packages/common/utils/src/object/object-keys.ts

```diff
@@ -1,6 +1,4 @@
 /** @tossdocs-ignore */
-export type ObjectKeys<T extends Record<PropertyKey, unknown>> = `${Exclude<keyof T, symbol>}`;
-
-export function objectKeys<Type extends Record<PropertyKey, unknown>>(obj: Type): Array<ObjectKeys<Type>> {
-  return Object.keys(obj) as Array<ObjectKeys<Type>>;
+export function objectKeys<T extends Record<PropertyKey, T[keyof T]>>(obj: T): Array<keyof T> {
+  return Object.keys(obj);
```

Comment:
I guess we could also modify and recycle existing ObjectKeys as shown below. 

```ts
export type ObjectKeys<T> = keyof T;
```

Of course, we could also define new ObjectValues.

```ts
export type ObjectValues<T> = T[keyof T];
```